tests/test_extensions/test_ext_intersphinx.py

```diff
@@ -551,30 +553,32 @@ def test_intersphinx_role(app, warning):
 
     app.build()
     content = (app.outdir / 'index.html').read_text(encoding='utf8')
-    wStr = warning.getvalue()
+    warnings = strip_colors(warning.getvalue()).splitlines()
+    index_path = Path(str(app.srcdir)) / 'index.rst'
```

Comment:
```suggestion
    index_path = app.srcdir / 'index.rst'
```

Reply:
ah yeh I guess this is ok now in the sphinx source; I'm always scared of what `app.srcdir` will be, after the whole v7.2.6 kerfuffle 😄